src/styling/MovieLibrary.css

```diff
@@ -0,0 +1,100 @@
+.library-container {
+  background-color: black;
```

Comment:
Really like how you've done black background and the wide padding to make a border.